be/src/common/config.h

```diff
@@ -922,6 +922,9 @@ CONF_String(aws_sdk_logging_trace_level, "trace");
 // Enabling RFC-3986 encoding will make sure these characters are properly encoded.
 CONF_Bool(aws_sdk_enable_compliant_rfc3986_encoding, "false");
 
+// use poco client to replace default curl client
+CONF_Bool(enable_poco_client_for_aws_sdk, "true");
```

Comment:
should this config as `false` by default? on-demand turn-on as experimental feature , and turn on by default later if stable enough